initializr-generator/src/main/java/io/spring/initializr/generator/buildsystem/maven/MavenBuildWriter.java

```diff
@@ -129,6 +131,22 @@ private void writeProperties(IndentingWriter writer, PropertyContainer propertie
 		});
 	}
 
+	private void writeDevelopers(IndentingWriter writer, MavenBuildSettings settings) {
```

Comment:
I do agree that this can further be improved. I am out of ideas at the moment but I would certainly need to dig deeper so I could possibly offer some suggestion to approach this problem.